CotEditor/Sources/EditorTextView.swift

```diff
@@ -837,6 +835,16 @@ final class EditorTextView: NSTextView, Themable {
         }
     }
     
+    /// whether replace tab with spaces
+    var isAutomaticTabExpansionEnabled: Bool {
+        didSet {
+            guard isAutomaticTabExpansionEnabled != oldValue else { return }
+            
+            // apply to view
+            self.invalidateDefaultParagraphStyle()
+        }
+    }
+    
     
```

Comment:
Updating the paragraph style after toggling `isAutomaticTabExpansionEnabled ` is not required. It doesn't affect to any style state but only behavior. So, I suppose you don't need to modify EditorTextView.swift. Just let it be.